File: ModelCS/Teacher.cs
using System.Collections.Generic;

namespace OOPRecords.Model
{
    public class Teacher
    {
        public virtual int Id { get; set; }

        public virtual string Title { get; set; }

        public virtual string LastName { get; set; }

        public virtual string JobTitle { get; set; }

        public virtual ICollection<Student> Tutees { get; set; } = new List<Student>();

        public override string ToString()
        {
            return $"{Title} {LastName}, {JobTitle}";
        }
    }
}

File: ModelVB/Teacher.vb
Public Class Teacher
    Public Overridable Property Id As Integer
    Public Overridable Property Title As String
    Public Overridable Property LastName As String
    Public Overridable Property JobTitle As String
    Private _tutees As ICollection(Of Student) = New List(Of Student)
    Public Overridable Property Tutees() As ICollection(Of Student)
        Get
            Return _tutees
        End Get
        Set(value As ICollection(Of Student))
            _tutees = value
        End Set
    End Property


    Public Overrides Function ToString() As String
        Return $"{Title} {LastName}, {JobTitle}"
    End Function
End Class
